packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-hydrate/table-hydrate-1k.benchmark.js

```diff
@@ -29,9 +29,7 @@ benchmark(`hydrate/table/hydrate/1k`, () => {
 
         const ssrHtml = renderComponent('benchmark-table', Table, props);
 
-        const fragment = new DOMParser().parseFromString(ssrHtml, 'text/html', {
-            includeShadowRoots: true,
-        });
+        const fragment = Document.parseHTMLUnsafe(ssrHtml);
```

Comment:
These SSR tests never worked properly for Firefox or Safari, since neither ever supported `includeShadowRoots`.

In any case, we mostly run our perf tests in Chrome.